tests/WE2E/setup_WE2E_tests.sh

```diff
@@ -42,43 +48,46 @@ function usage {
 machines=( hera jet cheyenne orion wcoss2 gaea odin singularity macos noaacloud )
 
 if [ "$1" = "-h" ] ; then usage ; fi
-[[ $# -le 1 ]] && usage
+[[ $# -le 2 ]] && usage
 
-machine=${1,,}
-account=$2
-compiler=${3:-intel}
-test_type=${4:-fundamental}
+homedir=$1
```

Comment:
This was changed per @danielabdi-noaa's comments; since this script is re-called automatically with homedir and it is not a user-supplied argument, the usage statement shouldn't include it